streamlit_web.py

```diff
@@ -57,6 +58,7 @@ def main():
                 "Summary",
                 "Topic Modeling",
                 "Interactive",
+                "Entity Analysis",
```

Comment:
I would move `Entity Analysis` up a little bit. This is also the order of how it's showing up in the streamlit sidebar.